include/modules/clock.hpp

```diff
@@ -41,6 +41,7 @@ class Clock final : public ALabel {
   const int cldMonColLen_{20};   // calendar month column length
   WS cldWPos_{WS::HIDDEN};       // calendar week side to print
   months cldCurrShift_{0};       // calendar months shift
+  int cldShift_{0};              // calendar months shift factor
```

Comment:
Should not be set to 1 by default?

Reply:
If you look at the commit I listed, the code used to initialize this to 0, so I guess I just copied that.
The Wiki does say the default is 1, and that makes sense, so I can update that.

Reply:
Actually I don't think this matters much, because given the structure of the code right now, this functionality isn't enabled unless the user explicitly puts a value into their config file. So really this "default" is never used.